forge-gui/res/cardsfolder/upcoming/dread_linnorm_scale_deflection.txt

```diff
@@ -0,0 +1,17 @@
+Name:Dread Linnorm
+ManaCost:6 G
+Types:Creature Giant
+PT:7/6
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.Self | ValidBlocker$ Creature.powerLE3 | Description$ CARDNAME can't be blocked by creatures with power 3 or less.
+AlternateMode:Adventure
+Oracle:Dread Linnorm can't be blocked by creatures with power 3 or less.
+
+ALTERNATE
+
+Name:Scale Deflection
+ManaCost:3 G
+Types:Instant Adventure
+A:SP$ PutCounter | Cost$ 3 G | ValidTgts$ Creature | TgtPrompt$ Select target creature | CounterType$ P1P1 | CounterNum$ 2 | SubAbility$ DBUntap | SpellDescription$ Put two +1/+1 counters on target creature and untap it.
```

Comment:
Cost$ and TgtPrompt$ not needed

Cost only needed if it does not match `ManaCost:`
Most simple `ValidTgts$` will auto-generate a good TgtPrompt

Reply:
Thank you, I used the script for Dragonscale Boon as a starting point:
```
Name:Dragonscale Boon
ManaCost:3 G
Types:Instant
A:SP$ PutCounter | Cost$ 3 G | ValidTgts$ Creature | TgtPrompt$ Select target creature | CounterType$ P1P1 | CounterNum$ 2 | SubAbility$ DBUntap | SpellDescription$ Put two +1/+1 counters on target creature and untap it.
SVar:DBUntap:DB$ Untap | Defined$ Targeted
DeckHas:Ability$Counters
Oracle:Put two +1/+1 counters on target creature and untap it.
```

I will submit a change for it as well.

Reply:
Many cards were scripted before the tweaks that made those unneeded... generally, unneeded stuff isn't harmful. But since most people learn scripting through looking at old cards, keeping things as clean and nice as possible (with tens of thousands of cards) is the way to go.